src/GlobalDiffEq.jl

```diff
@@ -25,6 +28,55 @@ function DiffEqBase.__solve(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase
     return sol
 end
 
-export GlobalRichardson
+struct GlobalAdjoint{A} <: GlobalDiffEqAlgorithm
+    alg::A
+end
+
+function DiffEqBase.__solve(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase.AbstractDAEProblem},
+                            alg::GlobalAdjoint, args...;
+                            dt,kwargs...)
+
+    opt = Dict(kwargs)
+    otheropts = delete!(copy(opt), :dt)
+    tstops = get(opt, :tstops, range(prob.tspan[1], stop=prob.tspan[2], step=dt))
+    local sol
+
+    sol_init = solve(prob, alg.alg, args...; dt=dt, adaptive=false, otheropts...)
+    n = length(prob.u0)
+    C = 11/6 # for k = 2, 2 random vectors
+    E₂ = 2/π
+    Eₙ = vec_expt_full(n)
+    rtol = get(opt, :reltol, 1e-3)
+    atol = get(opt, :abstol, 1e-6)
+    gtol = atol # target gtol is abstol? abstol cannot be zero?
```

Comment:
I am mainly setting gtol to atol for lack of another way to obtain gtol from the user. I was hesitant to add another keyword argument to the `solve()` equation. Do you have any recommendations on how to implement this? With respect to controlling rtol and atol in the original and adjoint solves, I have implemented this because rtol and atol are in the `otheropts` dict. However, this is likely over-complicated. I don't think I need to separate `dt` from the other `kwargs` for example. I will streamline how this is handled.